packages/pyright-internal/src/localization/package.nls.en-us.json

```diff
@@ -77,6 +77,8 @@
         "constructorNoArgs": "Expected no arguments to \"{type}\" constructor",
         "dataClassBaseClassFrozen": "A non-frozen class cannot inherit from a class that is frozen",
         "dataClassBaseClassNotFrozen": "A frozen class cannot inherit from a class that is not frozen",
+        "dataClassConverterFunction": "Argument of type \"{argType}\" is not an acceptable converter for field \"{fieldName}\" of type \"{fieldType}\"",
+        "dataClassConverterOverloads": "None of the overloads of \"{funcName}\" are acceptable converters for field \"{fieldName}\" of type \"{fieldType}\"",
```

Comment:
Grammar nit: Either change to "None ... _is_ acceptable" or (perhaps better) change to "No overloads ... are acceptable".

Also, the word "acceptable" strikes me as a bit weird in these errors. We don't use that term anywhere else. It sounds a bit condescending to my ears. Perhaps switch to "valid" or "invalid".